project/cmake/scripts/common/projectmacros.cmake

```diff
@@ -50,10 +50,32 @@ endfunction()
 # Copied from FindGTest.cmake 
 # Thanks to Daniel Blezek <blezek@gmail.com> for the GTEST_ADD_TESTS code
 function(GTEST_ADD_TESTS executable extra_args)
+    get_property(XBMC_GENERATED_TEST_FILES GLOBAL PROPERTY XBMC_GENERATED_TEST_FILES)
```

Comment:
@akva2 Was this the variable name you were referring to? I can probably rename it to something like `CORE_TEST_SUPPORT_FILES`